WorkloadManagementSystem/DB/JobDB.py

```
@@ -1480,6 +1480,34 @@ def getSiteMaskStatus( self ):
     return S_OK( siteDict )
 
 #############################################################################
+  def getAllSiteMaskStatus( self ):
+    """ Get the everything from site mask status
+    """
+    cmd = "SELECT Site,Status,LastUpdateTime,Author,Comment FROM SiteMask"
+
+    result = self._query( cmd )
+    siteDict = {}
+    if result['OK']:
```

you should return the error if the _query returns an error